packages/components/src/components/tooltip/tooltip.stories.ts

```diff
@@ -65,198 +65,62 @@ export const Default = {
 };
 
 /**
- * In case of having the content longer than the space available, the tooltip can be set to scrollable by overriding the `pointer-events` property to auto. In this case the padding-right should also be adjusted via `sd-tooltip::part(body)`. This behavior is heavily discouraged, but it is available in case of need.
+ * Use the `trigger` attribute to control how the tooltip is triggered.
  *
+ * - `click`: displays the tooltip when the trigger element is clicked
+ * - `hover`: displays the tooltip when the trigger element is hovered
+ * - `focus`: displays the tooltip when the trigger element is focused
+ * - `manual`: displays the tooltip when the `open` attribute is set to `true`
  */
-export const LongContent = {
-  parameters: {
-    controls: { exclude: 'content' }
-  },
-  render: () => {
-    return html` <div>
-      <style>
-        sd-tooltip::part(body) {
-          padding-right: 8px;
-        }
-        .long-content {
-          padding-right: 4px;
-        }
-        .long-content::-webkit-scrollbar {
-          width: 4px;
-        }
-
-        .long-content::-webkit-scrollbar-thumb {
-          background-color: transparent; /* make scrollbar transparent */
-          border-radius: 4px;
-        }
-
-        .long-content:hover::-webkit-scrollbar-thumb {
-          background-color: rgba(81, 81, 81, 0.9);
-        }
-      </style>
-      ${generateTemplate({
-        axis: {
-          x: {
-            type: 'slot',
-            name: 'content',
-            values: [
-              {
-                value: `<div class='long-content' slot='content' style='pointer-events:auto; width: 100px;'>Lorem ipsum sic semper dolor sit amet, consectetur adipiscing elit. Nulla euismod, nisl quis ultrices aliquam, nunc nisl aliquet nunc, quis aliquam nisl nisl quis nisl. Nulla euismod, nisl quis ultrices aliquam, nunc nisl aliquet nunc, quis aliquam nisl nisl quis nisl. Lorem ipsum sic semper dolor sit amet, consectetur adipiscing elit. Nulla euismod, nisl quis ultrices aliquam, nunc nisl aliquet nunc, quis aliquam nisl nisl quis nisl. Nulla euismod, nisl quis ultrices aliquam, nunc nisl aliqu et nunc, quis ali qu am nisl nisl quis nisl.</div>`,
-                title: 'long with fixed width'
-              }
-            ]
-          }
-        },
-        args: overrideArgs([
-          {
-            type: 'attribute',
-            name: 'open',
-            value: true
-          },
-          {
-            type: 'attribute',
-            name: 'placement',
-            value: 'bottom'
-          }
-        ]),
-        constants: [
-          {
-            type: 'template',
-            name: 'width',
-            value: '<div class="template-position">%TEMPLATE%</div>'
-          }
-        ]
-      })}
-    </div>`;
-  }
+export const Trigger = {
+  render: () => html` <sd-tooltip content="Lorem ipsum" placement="bottom" size="lg"></sd-tooltip> `
```

Comment:
```suggestion
  render: () => html` <sd-tooltip content="Lorem ipsum" placement="bottom" size="lg" trigger="click"></sd-tooltip> `
```

This might be very minor, but I noticed that the attribute we are describing is not visible under "show code", so I thought it would be more helpful to include it explicitly here even if it is the default.

![image](https://github.com/user-attachments/assets/e4832e69-3ae2-4f2e-b348-c1709cf4febd)